hyperspy/_signals/eds.py

```diff
@@ -710,9 +706,13 @@ def get_take_off_angle(self):
         tilt_stage = mp.Stage.tilt_alpha
         azimuth_angle = mp.Detector.EDS.azimuth_angle
         elevation_angle = mp.Detector.EDS.elevation_angle
+        if 'beta_tilt' not in mp:
```

Comment:
This test does not seems to work, it always return 0. 
I think it should be 'tilt_beta' but maybe it's only the case for emd files ?

Reply:
And replace mp by mp.Stage ?

Reply:
A last comment, an error shows up when value are not integers when i try this function. 
With all these modifications it works perfectly for me. Nice work !

Reply:
Do you want to open a PR to fix it?